repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java

```diff
@@ -268,29 +283,55 @@ private boolean createAuditIndex() throws IOException {
         Request request = new Request("PUT", INDEX_NAME);
         request.setEntity(entity);
         Response response = lowLevelClient.performRequest(request);
-        int statusCode = response.getStatusLine().getStatusCode();;
-        return statusCode == 200 ? true: false;
+        int statusCode = response.getStatusLine().getStatusCode();
+        return statusCode == 200 ? true : false;
     }
 
     private String getAuditIndexMappings() throws IOException {
-        String atlasHomeDir  = System.getProperty("atlas.home");
+        String atlasHomeDir = System.getProperty("atlas.home");
         String elasticsearchSettingsFilePath = (org.apache.commons.lang3.StringUtils.isEmpty(atlasHomeDir) ? "." : atlasHomeDir) + File.separator + "elasticsearch" + File.separator + "es-audit-mappings.json";
-        File elasticsearchSettingsFile  = new File(elasticsearchSettingsFilePath);
-        String jsonString  = new String(Files.readAllBytes(elasticsearchSettingsFile.toPath()), StandardCharsets.UTF_8);
+        File elasticsearchSettingsFile = new File(elasticsearchSettingsFilePath);
+        String jsonString = new String(Files.readAllBytes(elasticsearchSettingsFile.toPath()), StandardCharsets.UTF_8);
         return jsonString;
     }
 
-     private boolean checkIfIndexExists() throws IOException {
-         Request request = new Request("HEAD", INDEX_NAME);
-         Response response = lowLevelClient.performRequest(request);
-         int statusCode = response.getStatusLine().getStatusCode();;
-         if (statusCode == 200) {
-             LOG.info("Entity audits index exists!");
-             return true;
-         }
-         LOG.info("Entity audits index does not exist!");
-         return false;
-     }
+    private boolean checkIfIndexExists() throws IOException {
+        Request request = new Request("HEAD", INDEX_NAME);
+        Response response = lowLevelClient.performRequest(request);
+        int statusCode = response.getStatusLine().getStatusCode();
+        if (statusCode == 200) {
+            LOG.info("Entity audits index exists!");
+            return true;
+        }
+        LOG.info("Entity audits index does not exist!");
+        return false;
+    }
+
+    private boolean isFieldLimitDifferent() throws IOException {
+        JsonNode fieldLimit = getIndexFieldLimit();
+        Integer fieldLimitFromConfigurationFile = configuration.getInt(TOTAL_FIELD_LIMIT);
+        return fieldLimit == null || fieldLimitFromConfigurationFile.equals(fieldLimit.intValue());
+    }
+
+    private JsonNode getIndexFieldLimit() throws IOException {
```

Comment:
Add try/catch and don't throw an error instead just log it, as even if updating index limit fails, we would atlas to start up normally. If we will throw an error, the atlas won't start up.

Whatever we log as LOG.error, we receive a Sentry alert for that.